src/utils.ts

```diff
@@ -36,6 +36,33 @@ export namespace TsvUtils {
     });
     return rows.filter(notEmpty);
   };
+
+  export const parseObjectToTsv = (recordsObjects: any, headers: string[]) => {
```

Comment:
parse means reading a string, this is serialization / conversion

Reply:
correct, will rename this better :+1: